api/tests/core/educational/test_models.py

```diff
@@ -308,3 +312,219 @@ def test_collective_stock_has_reimbursed_collective_booking(self):
         stock: CollectiveStock = factories.CollectiveStockFactory.build()
         factories.ReimbursedCollectiveBookingFactory.build(collectiveStock=stock)
         assert not stock.is_cancellable_from_offerer
+
+
+class HasImageMixinTest:
+    def test_basic_methods(self):
+        class Image(HasImageMixin):
+            pass
+
+        image = Image()
+        image.id = 1
+        image.imageId = 25
+        image.imageCrop = {}
+        image.imageCredit = "toto"
+        image.hasOriginal = True
+
+        assert image.imageId != None
+        assert image.imageUrl == "http://localhost/storage/thumbs/image/1_25.jpg"
+        assert image.imageOriginalUrl == "http://localhost/storage/thumbs/image/1_25_original.jpg"
+        assert image._get_image_storage_id() == "image/1_25.jpg"
+        assert image._get_image_storage_id(original=True) == "image/1_25_original.jpg"
+        assert image._get_image_storage_id(original=True, image_id=666) == "image/1_666_original.jpg"
+
+    @mock.patch("pcapi.core.educational.models.store_public_object")
+    @mock.patch("pcapi.core.educational.models.delete_public_object")
+    @mock.patch("pcapi.core.educational.models.process_original_image", return_value=b"tutu")
+    @mock.patch("pcapi.core.educational.models.standardize_image", return_value=b"tata")
```

Comment:
(détail) Pour des tests un peu plus clairs : 

```suggestion
    @mock.patch("pcapi.core.educational.models.process_original_image", return_value=b"processed original image contents")
    @mock.patch("pcapi.core.educational.models.standardize_image", return_value=b"standardized image contents")
```

(et idem pour les autres tests)